### DISCUSSION TITLE
Grading

### DISCUSSION THREAD

**Discussion by ryankong123** [chulminy/CIVE497-CIVE700#137] [General]
Just curious, what is the grade distribution for the assignments? Are they all worth the same?

**Comment by chulminy** [comment]
I will just add all scores that you receive and divide it by the sum of perfect marks. 

It's not the same weight distribution of each task because a full mark for Task 6 is 150.

**Reply by ryankong123** [reply]
What are the marks for Task 08?

**Reply by chulminy** [reply]
150

**Reply by ryankong123** [reply]
Thanks for the info
